docs/guide/cloud_concepts/cli.md

```diff
@@ -0,0 +1,371 @@
+# CLI
+
+In conjunction with both the GraphQL API and the UI, Prefect offers a fully integrated CLI for working with Prefect Cloud. With the GraphQL API being the backbone for communicating with Prefect Cloud, we have designed the CLI to operate as a wrapper for GraphQL queries and mutations.
+
+In this current state the Prefect CLI functions mainly as a way to read Prefect Cloud metadata as well as a lightweight way to perform some more functional tasks such as running a flow that was deployed to Cloud.
```

Comment:
```suggestion
In this current state the Prefect CLI functions mainly as a way to read Prefect Cloud metadata as well as a lightweight way to perform some more functional operations such as running a flow that was deployed to Cloud.
```
to avoid confusion with Prefect Tasks